src/util/string_utils.js

```diff
@@ -10,6 +10,11 @@ const email_regexp = /^((([a-z]|\d|[!#\$%&'\*\+\-\/=\?\^_`{\|}~]|[\u00A0-\uD7FF\
 const escape_regexp = /[\\^$.*+?()[\]{}|]/g;
 const access_key_regexp = /^[a-zA-Z0-9]{20}$/;
 const secret_key_regexp = /^[a-zA-Z0-9+/]{40}$/;
+// regex for IAM service
+const AWS_IAM_PATH_REGEXP = /(\u002F)|(\u002F[\u0021-\u007E]+\u002F)/;
+const AWS_USERNAME_REGEXP = /[\w+=,.@-]+/;
+const AWS_LIST_MARKER = /[\u0020-\u00FF]+/;
```

Comment:
unclear. why is this the range?

Reply:
I took the pattern for the docs, for example ([ListUsers](https://docs.aws.amazon.com/IAM/latest/APIReference/API_ListUsers.html#API_ListUsers_RequestParameters)):

> Pattern: [\u0020-\u00FF]+

It was hard to find something invalid in this pattern - the example I found it:

> const invalid_characters = '\uD83D\uDE0A'; //emoji is not part of the pattern

Reply:
Perhaps this is very specific for list users, so better rename this to AWS_LIST_USERS_MARKER

Reply:
It is for list users and list access keys.
So I will suggest renaming it to `AWS_IAM_LIST_MARKER`, WDYT?

Reply:
sounds great

Reply:
I will change it also, thanks.